src/index.js

```diff
@@ -747,6 +747,35 @@ walletRouter.post(
   }
 );
 
+/**
+ * GET request to obtain adress information
+ * For the docs, see api-docs.js
+ */
+ walletRouter.get(
+  '/address-info',
+  query('address').isString(),
+  query('token').isString().optional(),
+  async (req, res) => {
+    try {
```

Comment:
Why do we have this try? Which exception are we expecting to handle?

Reply:
If the specified address does not belong to the wallet an error will be fired. To correctly respond to the api caller, this try catch statement catches the error and format the return.

Reply:
Maybe we should capture the specific error

Reply:
Exactly. We must avoid capturing generic exceptions at all costs.

Reply:
Done, thanks!